<?xml version="1.0" encoding="UTF-8"?><testrun name="Rerun org.pris2.polinomio.PolinomioTest.testPolinomio" project="pris2" tests="1" started="1" failures="0" errors="1" ignored="0">
  <testcase name="testPolinomio" classname="org.pris2.polinomio.PolinomioTest" time="0.0">
    <error>java.lang.IndexOutOfBoundsException: Index: 5, Size: 5&#13;
	at java.util.ArrayList.rangeCheck(Unknown Source)&#13;
	at java.util.ArrayList.get(Unknown Source)&#13;
	at org.pris2.polinomio.polinomio.agruparMonomios(polinomio.java:61)&#13;
	at org.pris2.polinomio.PolinomioTest.testPolinomio(PolinomioTest.java:41)&#13;
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)&#13;
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)&#13;
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)&#13;
	at java.lang.reflect.Method.invoke(Unknown Source)&#13;
	at junit.framework.TestCase.runTest(TestCase.java:176)&#13;
	at junit.framework.TestCase.runBare(TestCase.java:141)&#13;
	at junit.framework.TestResult$1.protect(TestResult.java:122)&#13;
	at junit.framework.TestResult.runProtected(TestResult.java:142)&#13;
	at junit.framework.TestResult.run(TestResult.java:125)&#13;
	at junit.framework.TestCase.run(TestCase.java:129)&#13;
	at org.eclipse.jdt.internal.junit.runner.junit3.JUnit3TestReference.run(JUnit3TestReference.java:131)&#13;
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:459)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:675)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:382)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:192)&#13;
&#13;
</error>
  </testcase>
</testrun>
